src/multicloud-connector/HISTORY.rst

```diff
@@ -0,0 +1,8 @@
+.. :changelog:
+
+Release History
+===============
+
+1.0.0b1
+++++++
+* Initial release.
```

Comment:
`test-permission` command,

1. The output `{}` doesn't make sense. 

![image](https://github.com/user-attachments/assets/1d271ccd-837d-499c-b46c-e42623f97d2e)

2. It's NOT working.

CLI output
![image](https://github.com/user-attachments/assets/35f56f62-0d97-46c8-8b2a-b83ea91ed695)

Portal output
![image](https://github.com/user-attachments/assets/76366aa0-8a22-4c6a-8091-5eef21c4d824)

3. It's unclear what does `--ids` represent. Is it the connector Id or something else. Please treat this as a global comment as it's there in all the commands.

![image](https://github.com/user-attachments/assets/946bc93f-4f95-4387-8933-00f64f95d5fd)

4. It's unclear what `--no-wait` does. I don't see any difference of behavior with "0 or 1". Looks like the CLI command is not executing as I don't see the response `{}`

![image](https://github.com/user-attachments/assets/cc0e55cc-c063-4301-861c-57c451bc3ed4)